pkg/blueprint/blueprint_test.go

```diff
@@ -23,16 +23,16 @@ version = "2.4.*"
 
 [[customizations.filesystem]]
 mountpoint = "/var"
-size = 2147483648
+minsize = 2147483648
```

Comment:
I assume there is no risk here that there are "wrong" blueprint tomls out there that we would break with this change?